src/server/dash-server.ts

```diff
@@ -33,6 +34,7 @@ export class DashServer {
     const app = express();
     const litPath = path.dirname(require.resolve('lit-html'));
 
+    app.use(enforceHTTPS);
```

Comment:
How about conditionally load this middleware, instead or the middleware becoming a noop? Seems more clear.